include/zephyr/xen/public/arch-arm.h

```diff
@@ -436,8 +436,8 @@ typedef uint64_t xen_callback_t;
 
 #define GUEST_RAM_BANKS                        2
 
-#define GUEST_RAM0_BASE                        xen_mk_ullong(0x40000000) /* 3GB of low RAM @ 1GB */
```

Comment:
Actually, these values are in sync with original amount in mainline public headers (since it was upstreamed) - https://github.com/xen-project/xen/blob/stable-4.17/xen/include/public/arch-arm.h#L454

Our Xen is customized due to same reasons, specifically this part by this commit - https://github.com/firscity/xen/commit/2d5836a64efc332e3a28bf982af2b204b7bae0ba

So, I think it is worth to reword commit message and explain why we have to change these values for zephyr-v3.3.0-xt branch

Reply:
actually I wrote in a commit message why we need it

Reply:
ohh, sorry, I don't understand your point correctly
no problem, I'll fix it

Reply:
added information